components/01-atoms/text/text/07-byline.twig

```diff
@@ -0,0 +1,35 @@
+{#
+/**
+ * Available variables:
+ * - byline_base_class - the base classname
```

Comment:
After chatting with Brian and JoeT about some of the variable documentation we've been doing we've decided that it's not worth add comments for `*__base_class`, `*__modifiers`, and `*__blockname`. These variables should be a given with any component as we use BEM class structure for everything. Can you remove these comments?